engine/src/flutter/lib/web_ui/lib/src/engine/safe_browser_api.dart

```diff
@@ -121,38 +62,11 @@ num? parseFontSize(DomElement element) {
 /// Provides haptic feedback.
 void vibrate(int durationMs) {
   final DomNavigator navigator = domWindow.navigator;
-  if (hasJsProperty(navigator, 'vibrate')) {
-    js_util.callMethod<void>(navigator, 'vibrate', <num>[durationMs]);
+  if (navigator.hasProperty('vibrate'.toJS).toDart) {
+    navigator.callMethod('vibrate'.toJS, durationMs.toJS);
```

Comment:
Consider adding this as a method to `DomNavigator`.

Reply:
ack